handler/src/main/java/io/netty/handler/ssl/Ciphers.java

```diff
@@ -0,0 +1,763 @@
+
+    /**
+     * TLS_ECDHE_PSK_WITH_AES_128_CBC_SHA
+     */
+    public static final String TLS_ECDHE_PSK_WITH_AES_128_CBC_SHA = "TLS_ECDHE_PSK_WITH_AES_128_CBC_SHA";
+
+    /**
+     * TLS_SRP_DSS_WITH_AES_128_CBC_SHA
+     */
+    public static final String TLS_SRP_DSS_WITH_AES_128_CBC_SHA = "TLS_SRP_DSS_WITH_AES_128_CBC_SHA";
+
+    /**
+     * TLS_SRP_RSA_WITH_AES_128_CBC_SHA
+     */
+    public static final String TLS_SRP_RSA_WITH_AES_128_CBC_SHA = "TLS_SRP_RSA_WITH_AES_128_CBC_SHA";
+
+    /**
+     * TLS_SRP_WITH_AES_128_CBC_SHA
+     */
+    public static final String TLS_SRP_WITH_AES_128_CBC_SHA = "TLS_SRP_WITH_AES_128_CBC_SHA";
+
+    /**
+     * TLS_RSA_PSK_WITH_AES_128_CBC_SHA256
+     */
+    public static final String TLS_RSA_PSK_WITH_AES_128_CBC_SHA256 = "TLS_RSA_PSK_WITH_AES_128_CBC_SHA256";
+
+    /**
+     * TLS_DHE_PSK_WITH_AES_128_CBC_SHA256
+     */
+    public static final String TLS_DHE_PSK_WITH_AES_128_CBC_SHA256 = "TLS_DHE_PSK_WITH_AES_128_CBC_SHA256";
+
+    /**
+     * TLS_RSA_PSK_WITH_AES_128_CBC_SHA
+     */
+    public static final String TLS_RSA_PSK_WITH_AES_128_CBC_SHA = "TLS_RSA_PSK_WITH_AES_128_CBC_SHA";
+
+    /**
+     * TLS_DHE_PSK_WITH_AES_128_CBC_SHA
+     */
+    public static final String TLS_DHE_PSK_WITH_AES_128_CBC_SHA = "TLS_DHE_PSK_WITH_AES_128_CBC_SHA";
+
+    /**
+     * TLS_ECDHE_PSK_WITH_CAMELLIA128_SHA256
+     */
+    public static final String TLS_ECDHE_PSK_WITH_CAMELLIA128_SHA256 = "TLS_ECDHE_PSK_WITH_CAMELLIA128_SHA256";
+
+    /**
+     * TLS_RSA_PSK_WITH_CAMELLIA128_SHA256
+     */
+    public static final String TLS_RSA_PSK_WITH_CAMELLIA128_SHA256 = "TLS_RSA_PSK_WITH_CAMELLIA128_SHA256";
+
+    /**
+     * TLS_DHE_PSK_WITH_CAMELLIA128_SHA256
+     */
+    public static final String TLS_DHE_PSK_WITH_CAMELLIA128_SHA256 = "TLS_DHE_PSK_WITH_CAMELLIA128_SHA256";
+
+    /**
+     * TLS_RSA_WITH_AES_128_CBC_SHA
+     */
+    public static final String TLS_RSA_WITH_AES_128_CBC_SHA = "TLS_RSA_WITH_AES_128_CBC_SHA";
+
+    /**
+     * TLS_RSA_WITH_SEED_SHA
+     */
+    public static final String TLS_RSA_WITH_SEED_SHA = "TLS_RSA_WITH_SEED_SHA";
+
+    /**
+     * TLS_RSA_WITH_CAMELLIA128_SHA
+     */
+    public static final String TLS_RSA_WITH_CAMELLIA128_SHA = "TLS_RSA_WITH_CAMELLIA128_SHA";
+
+    /**
+     * TLS_RSA_WITH_IDEA_CBC_SHA
+     */
+    public static final String TLS_RSA_WITH_IDEA_CBC_SHA = "TLS_RSA_WITH_IDEA_CBC_SHA";
+
+    /**
+     * TLS_PSK_WITH_AES_128_CBC_SHA256
+     */
+    public static final String TLS_PSK_WITH_AES_128_CBC_SHA256 = "TLS_PSK_WITH_AES_128_CBC_SHA256";
+
+    /**
+     * TLS_PSK_WITH_AES_128_CBC_SHA
+     */
+    public static final String TLS_PSK_WITH_AES_128_CBC_SHA = "TLS_PSK_WITH_AES_128_CBC_SHA";
+
+    /**
+     * TLS_PSK_WITH_CAMELLIA128_SHA256
+     */
+    public static final String TLS_PSK_WITH_CAMELLIA128_SHA256 = "TLS_PSK_WITH_CAMELLIA128_SHA256";
+
+    public static Iterable<String> of(String... ciphers) {
```

Comment:
Honestly I am not sure if this class is really useful at all... I would prefer to just not add this honestly. @chrisvest WDYT ?